common/management/commands/create_test_data.py

```diff
@@ -1754,47 +1855,65 @@ def _create_applications(
     users: list[User],
     age_groups: list[AgeGroup],
     reservation_purposes: list[ReservationPurpose],
-    reservation_units: list[ReservationUnit],
     cities: list[City],
     *,
     number: int = 20,
 ) -> list[Application]:
+    now = datetime.now(tz=UTC)
+
     contact_persons = _create_persons(number=number)
     billing_addresses = _create_addresses(number=number)
     organisations = _create_organisation(billing_addresses)
+
     applications: list[Application] = []
-    now = datetime.now(tz=UTC)
 
-    items: zip[tuple[Person, Address, tuple[str, Organisation | None]]]
-    items = zip(contact_persons, billing_addresses, organisations, strict=True)
-    for contact_person, billing_address, (applicant_type, organisation) in items:
-        # User is the overall admin mode often, but other users are also possible
-        weights = [len(users)] + ([1] * (len(users) - 1))
-        user: User = weighted_choice(users, weights=weights)
-
-        # No application for future application rounds
-        application_round: ApplicationRound = random.choice(application_rounds)
-        while application_round.application_period_begin > now:
-            application_round = random.choice(application_rounds)
-
-        application = Application(
-            applicant_type=applicant_type,
-            contact_person=contact_person,
-            user=user,
-            organisation=organisation,
-            application_round=application_round,
-            billing_address=billing_address,
-            home_city=random.choice(cities),
-            additional_information=faker_fi.sentence(),
-        )
-        applications.append(application)
+    # The first past and present application rounds have "a lot" of applications
+    application_counts = iter([200, 200] + ([number] * (len(application_rounds) - 2)))
+
+    for application_round in application_rounds:
+        # Create N application per application round
+        for _ in range(next(application_counts)):
+            # No application for future application rounds
+            if application_round.application_period_begin > now:
+                break
+
+            # User is the overall admin mode often, but other users are also possible
+            weights = [len(users)] + ([1] * (len(users) - 1))
+            user = weighted_choice(users, weights=weights)
+
+            applicant_type, organisation = random.choice(organisations)
+
+            # 2/3 of applications have been sent
+            # For open application rounds, this means application is a draft
```

Comment:
The comment can be clarified slightly:
_2/3 of applications have been sent **by the user**
For open application rounds, this means application **has been received for handling (and is not a draft)**
For past application rounds, this means application is sent **and handled** (and not expired)_

Reply:
Fixed comment.